### DISCUSSION TITLE
How to read and change the value of editor

### DISCUSSION THREAD

**Discussion by Adkimsm** [Milkdown/milkdown#1088] [Q&A] [answered]
I'm using react, is there a way to read and change the editor values?
My project is at https://github.com/adkimsm/milkdown-editor

**Comment by Adkimsm** [comment]
I have already found I can use listener to read the value,but I have not found how to change it.

**Reply by Saul-Mirone** [reply, marked as answer]
Are you looking for [replaceAll macro](https://milkdown.dev/docs/api/utils#replace-all-markdown-string-flush-boolean-false-fn-ctx-ctx)?

**Reply by Adkimsm** [reply]
yeah,that is it.thank you.
